### PR TITLE
Add RBAC for streaming app [SAME VERSION]

### PR DESCRIPTION
**Pull request by jiria**
**What this PR does / why we need it**:
Adds necessary RBAC rules for the streaming app

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] added code adheres to standard Rust formatting (`cargo fmt`)
- [ ] code builds properly (`cargo build`)
- [ ] code is free of common mistakes (`cargo clippy`)
- [ ] all Akri tests succeed (`cargo test`)
- [ ] inline documentation builds (`cargo doc`)
- [ ] version has been updated appropriately (`./version.sh`)

### CODE REVIEW

File: deployment/samples/akri-video-streaming-app.yaml [hunk 1]
```diff
@@ -47,3 +48,30 @@ spec:
...
+metadata:
+  name: akri-video-streaming-app-binding
+roleRef:
+  apiGroup: ""
+  kind: ClusterRole
+  name: akri-video-streaming-app-role
+subjects:
+  - kind: ServiceAccount
+    name: akri-video-streaming-app-sa
+    namespace: default
```
Comment (jiria) [inline deployment/samples/akri-video-streaming-app.yaml:77]: First time creating rbac rules. @bfjelds do you know if this will work for all namespaces?

**Review by bfjelds — APPROVED** [review]

**Review by kate-goldenring — APPROVED** [review]

File: deployment/samples/akri-video-streaming-app.yaml (hunk 1 above)
Comment (bfjelds) [reply deployment/samples/akri-video-streaming-app.yaml:77]: i omitted the namespace in the helm rbac

Comment (bfjelds) [reply deployment/samples/akri-video-streaming-app.yaml:77]:
```yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: akri-agent-sa
```

Comment (bfjelds) [reply deployment/samples/akri-video-streaming-app.yaml:77]: oops ... wrong section

the helm stuff uses the helm install namespace:
```yaml
subjects:
  - kind: 'ServiceAccount'
    name: 'akri-controller-sa'
    namespace: {{ .Release.Namespace }}
```

Comment (jiria) [reply deployment/samples/akri-video-streaming-app.yaml:77]: I omitted it in the ServiceAccount as well, but it did not let me drop it in the subjects.

Comment (jiria) [reply deployment/samples/akri-video-streaming-app.yaml:77]: Ok, perhaps can be resolved in the integrate to Helm PR.

**Pull request merged**